documents/policies/Competition Requirements.md

```diff
@@ -0,0 +1,91 @@
+# WCA Competition Requirements Policy
+<div class="version">
+### Version 1.0
+</div>
+
+## Purpose
+The purpose of this policy is to define the requirements and processes that a competition must meet and follow in order to be recognized as a WCA Competition.
+
+## Policy
+### WCA Competitions
+An official WCA Competition must:
+
+1. Be announced at least 28 days (four weeks) before the start of the competition on the WCA website.
```

Comment:
```suggestion
1. Be announced at least 28 days (four weeks) before midnight of the start of the competition, measured in UTC, on the WCA website.
```
It should be clarified that we use the UTC time to determine the calendar and not the local time. Here's my attempt. This matches what the WCAT sees for __ days away when reviewing a competition.

Reply:
Suggestion: "midnight preceding" is less ambiguous than "midnight of".

Reply:
I'm adding this to the new version I'm working on, using 00:00 UTC to try not to be ambiguous.